src/shared/types/component.enum.ts

```diff
@@ -0,0 +1,5 @@
+export const EComponent = {
```

Comment:
export enum EComponet {
  REST_APPLICATION = 'RestApplication'
}

export const COMPONENTS = {
  REST_APPLICATION: Symbol.for()
}